### PR TITLE
chore: add name and description to analytic events

### PR DESCRIPTION
**Pull request by rephus**
Closes: #11283

### Description:

Screenshot from 2024-09-24 09-50-22
Screenshot from 2024-09-24 09-50-57
Screenshot from 2024-09-24 09-51-39

### Reviewer actions

- [ ] I have manually tested the changes in the preview environment
- [ ] I have reviewed the code
- [ ] I understand that "request changes" will block this PR from merging

### CODE REVIEW

**Comment by TuringLovesDeathMetal** [comment]
@rephus I can't see `description` in the image, but I'm assuming you've added it?

**Comment by rephus** [comment]
> @rephus I can't see `description` in the image, but I'm assuming you've added it?

Yes, I did, you can see it on the first 2 images, on the third I guess it was not captured in the screenshot

File: packages/backend/src/analytics/LightdashAnalytics.ts [hunk 1]
```diff
@@ -335,6 +335,8 @@ type RollbackChartVersionEvent = BaseTrack & {
 export type CreateSavedChartVersionEvent = BaseTrack & {
     event: 'saved_chart_version.created';
     properties: {
+        name: string;
```
Comment (andrewdw) [inline packages/backend/src/analytics/LightdashAnalytics.ts:338]: Should be 'title'

File: packages/backend/src/services/SavedChartsService/SavedChartService.ts [hunk 2]
```diff
@@ -184,6 +184,8 @@ export class SavedChartService extends BaseService {
                 : undefined;
 
         return {
+            name: savedChart.name,
```
Comment (andrewdw) [inline packages/backend/src/services/SavedChartsService/SavedChartService.ts:187]: should be 'title'

**Review by andrewdw — CHANGES_REQUESTED** [review]

**Review by andrewdw — APPROVED** [review]

**Pull request merged**